src/simsopt/geo/boozersurface.py

```diff
@@ -51,7 +52,7 @@ def __init__(self, biotsavart, surface, label, targetlabel):
     def recompute_bell(self, parent=None):
         self.need_to_run_code = True
 
-    def boozer_penalty_constraints(self, x, derivatives=0, constraint_weight=1., scalarize=True, optimize_G=False):
+    def boozer_penalty_constraints(self, x, derivatives=0, constraint_weight=1., scalarize=True, optimize_G=False, weight_inv_modB=False):
```

Comment:
Would it make sense for the default of weight_inv_modB to be True? I would think that option would be more robust.